test/addons/rbd-mirror/start

```diff
@@ -106,6 +106,10 @@ def configure_rbd_mirroring(cluster, peer_info):
         context=cluster,
     )
 
```

Comment:
Missing log for applying this file - this was logged before for the entire kustomization.

Reply:
log added